en/docs/learn/configuring-oidc-federated-idp-initiated-logout.md

```diff
@@ -0,0 +1,188 @@
+# Configuring OIDC Federated IdP Initiated Logout
+
+WSO2 Identity Server (WSO2 IS) supports handling logout requests from OIDC federated identity providers. When an OIDC
+logout request is received by the OIDC federated identity provider, WSO2 IS processes the request, terminates the
+sessions and then responds to the identity provider.
+
+## Scenario
+
+The diagram below illustrates the OIDC federated identity provider initiated logout scenario. **WSO2 IS** and
+**Application 2** are configured as service providers in the **OIDC provider**. **OIDC provider** is configured as an
+identity provider, and **Application 1** is configured as a service provider in the **WSO2 IS**. When the user initiates
+the logout from **Application 2**, first the federated **OIDC provider** handles the request and propagates the logout
+request to **WSO2 IS**. After receiving the logout request from the federated identity provider, **WSO2 IS** processes
+the request and terminates the sessions and sends back a valid logout response. User is logged out from the
```

Comment:
```suggestion
the request and terminates the sessions and sends back a logout response. User is logged out from the
```

Reply:
Add the below to a note section

> If logout is successful in WSO2IS, it will respond with 200 OK. If there are any server-side errors or client errors(Eg: claim validation fails), it will respond with Bad request.

_